src/dosbox.cpp

```diff
@@ -796,6 +796,20 @@ void DOSBOX_Init()
 	                  "On 'auto' the mode is determined by 'sbtype'.\n"
 	                  "All OPL modes are AdLib-compatible, except for 'cms'.");
 
+	pstring = secprop->Add_string("opl_fadeout", when_idle, "off");
+	pstring->Set_help(
+	        "Fadeout the OPL synth output after the last IO port write:\n"
+	        "  off:       Don't fadeout; residual output will play forever (default).\n"
+	        "  on:        Wait 350 ms before fading out over a 350 ms period.\n"
+	        "  <custom>:  A custom fadeout definition in the following format:\n"
+	        "               WAIT FADE\n"
+	        "             Where WAIT is how long (in milliseconds) before fading begins,\n"
+	        "             and FADE is the duration (in milliseconds) of the fade out period.\n"
+	        "             Both values can range from 100 to 10000 (milliseconds).\n"
+	        "             Examples:\n"
+	        "                 1000 3000  (Wait 1 s before fading-out over a 3 s period)\n"
```

Comment:
1s, 3s, 10ms etc. would look better IMO (no space).

Reply:
Ok; but the International System of Units (SI) always has a space between values and their units - https://physics.nist.gov/cuu/Units/checklist.html

Reply:
Yeah, OK, you're right. Just "over a 3 s" seemed funky, but that's what you get with monospaced fonts 😅 

We use "70 Hz" in other places, but "256K" and "8MB" in other places, so it's a bit all over the place. Sticking "K" and "MB" to the numbers is kind of a "convention" I guess 😄

Reply:
(I squished 'em anyway because it's more consistent w/ the conf :grinning: )